src/flb_log_event_decoder.c

```diff
@@ -70,7 +70,7 @@ void flb_log_event_decoder_reset(struct flb_log_event_decoder *context,
     context->offset = 0;
     context->buffer = input_buffer;
     context->length = input_length;
-    context->decode_result = FLB_EVENT_DECODER_ERROR_NO_INCOMING_DATA;
+    context->last_result = FLB_EVENT_DECODER_ERROR_NO_INCOMING_DATA;
```

Comment:
Please replace `FLB_EVENT_DECODER_ERROR_NO_INCOMING_DATA` with `FLB_EVENT_DECODER_ERROR_INSUFFICIENT_DATA`.

Reply:
I replaced it.